frames/src/main/java/com/checkout/frames/cvvinputfield/style/DefaultCVVInputFieldStyle.kt

```diff
@@ -0,0 +1,13 @@
+package com.checkout.frames.cvvinputfield.style
+
+import com.checkout.frames.style.component.base.InputFieldIndicatorStyle
+import com.checkout.frames.style.component.base.InputFieldStyle
+import com.checkout.frames.style.component.default.DefaultLightStyle
+
+public object DefaultCVVInputFieldStyle {
```

Comment:
why we need this layer? `DefaultCVVInputFieldStyle` instead of using the default `InputFieldStyle`

Reply:
Default inputfieldstyle has border and custom coloring from default payment form style. However, in this case we dont need to provide border (same as default basictextfield).